SourcingTests/FetchedResultsDataProviderTests.swift

```diff
@@ -216,14 +216,25 @@ class FetchedResultsDataProviderTests: XCTestCase {
         XCTAssert(didUpdateDataSource)
     }
     
-    func testProcessUpdatesNumberForMoveChange() {
+    func testProcessUpdatesForMoveChange() {
         //Given
-        var updatesNumber = 0
+        var updateIndexPath = IndexPath()
+        var moveIndexPaths = [IndexPath]()
         
         dataProvider = try! FetchedResultsDataProvider(fetchedResultsController: fetchedResultsController, dataProviderDidUpdate: { _ in })
         dataProvider.whenDataProviderChanged = { updates in
-            if !updates!.isEmpty {
-                updatesNumber += 1
+            if updates!.isEmpty {
+                return
+            }
+                
+            for update in updates! {
```

Comment:
can you replace ` for update in updates!` with ` for update in updates ?? []`. This will cause the test to fail instead of crashing when updates is nil

Reply:
you're right - I will fix it today